issue_08/script.js

```diff
@@ -0,0 +1,49 @@
+function startLoading() {
+    const loadingPlace = document.getElementById('js-loading');
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    loadingPlace.appendChild(loadingImg);
+}
+
+function stopLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function createLists(resolvedData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+
+    for (const item of resolvedData) {
+        const li = document.createElement('li');
+        const img = document.createElement('img');
+
+        li.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+        fragment.appendChild(li).appendChild(img);
+    }
+    ul.appendChild(fragment);
+}
+
+const getListData = new Promise((reject) => {
+    startLoading();
+    const listData = [{
+        img: "1.png",
+        alt: "画像1",
+        text: "ブックマーク"
+    }, {
+        img: "2.png",
+        alt: "画像2",
+        text: "メッセージ"
+    }];
+    setTimeout(() => {
+        reject(Error('エラーです'));
+    }, 3000);
+});
+getListData.catch((error) => {
+    return error;
+});
+getListData.then((catched) => {
+    console.error(catched);
+});
```

Comment:
```suggestion
getListData.catch((error) => {
    console.error(error);
});
```
I don't need .then's processing because it's rejected.

Reply:
Fixed:sparkles:
https://github.com/yoshikawa-akr/js-lessons/pull/8/commits/6751372ee9a83cf58c3573ea5d29ea993124327f

---
The reason was that I had not set `resolve` as a Promise argument, and `reject` was recognized as the first argument. Therefore, in the previous code, `catch` was not processed and only `then` was processed.

Reply:
I should have asked a question about the first code😭🙏

Thank you for teaching me!

I learned a lot✨